tcping.go

```diff
@@ -317,6 +319,11 @@ func processUserInput(tcpStats *stats) {
 	tcpStats.userInput.ip = resolveHostname(tcpStats)
 	tcpStats.startTime = time.Now()
 	tcpStats.userInput.probesBeforeQuit = *probesBeforeQuit
+	tcpStats.userInput.intervalBetweenProbes = secondsToDuration(*secondsBetweenProbes)
+	if tcpStats.userInput.intervalBetweenProbes < 2*time.Millisecond {
+		tcpStats.printer.printError("Wait interval should be more than 2 ms")
```

Comment:
I am not sure if I understand the reason for this check. 

Could you elaborate it for me please?

Reply:
I just took that limit from unix ping 

> Only super-user may set interval to values less than 2 ms.
from man ping.
If you think this check is unnecessary I'll just remove it.

Reply:
ah interesting. I did not know about it.

It is fine for now.